frontend/device/input.lua

```diff
@@ -449,6 +458,29 @@ function Input:handleTouchEvPhoenix(ev)
     end
 end
 
+function Input:handleOrientationEv(ev)
+
+    if ev.type == EV_ABS then
+        if ev.code == ABS_PRESSURE and self.device.screen:getScreenMode() == 'portrait' then
+            local refreshUI = false
+            local rotation_mode = 0
+            if ev.value == DEVICE_ORIENTATION_PORTRAIT then
+                refreshUI = true
+            elseif ev.value == DEVICE_ORIENTATION_PORTRAIT_ROTATED then
+                refreshUI = true
+                rotation_mode = 2
+            end
+
+            local oldRotation = self.device.screen:getRotationMode()
+            if refreshUI and rotation_mode ~= oldRotation then
+                self.device.screen:setRotationMode(rotation_mode)
+                local UIManager = require("ui/uimanager")
+                UIManager:onRotation()
```

Comment:
Actually is it a good idea to reference UIManager in Input.lua?

Anyone can fix this please, not sure what is the better way.